rero_ebooks/mappings/v7/ebooks/ebook-v1.0.0.json

```diff
@@ -0,0 +1,7976 @@
+{
+  "settings": {
```

Comment:
what do we need to be indexed in this instance?

Reply:
This is the standard marc21 mapping from invenio. Mappings are necessary for declaring `OAI sets`, searching in the API and search in the OAI requests.